localscripts/generate-po2json.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+[ -x "$(command -v npm)" ] || { >&2 echo "npm is required but not installed, exiting."; exit 1; }
```

Comment:
Here I assume we will have the nodejs installed 🤔

Reply:
Should work, it's run by the `p4-builder` image which has npm on it, see https://github.com/greenpeace/planet4-handbook/blob/e30c3226dee25cf9e67664c2dbb34480a0b9ed16/.circleci/config.yml#L11 . The builder has access to this script because it checks out the handbooks code.